test/specs/Class5/Homework-2.js

```diff
@@ -24,4 +24,108 @@
  * 5. Click on Done button
  * 6. Verify total-travalers is equals to the number of adults mentioned in rooms
  * 
- */
\ No newline at end of file
+ */
+
+const { expect } = require("chai");
+const { describe, it } = require("mocha");
+const moment = require("moment");
+
+describe('Homework-2', async () => {
+
+    // TC-1: Verify the current date is select by default in Sign Up dropdown
+
+    it('Verify the current date is select by default in Sign Up dropdown', async () => {
+
+        //1. Launch facebook.com
+
+        await browser.url('https://www.facebook.com/');
+        await browser.pause(2000);
+
+        //2. Click on Create New Account button
+
+        const createAccLocator = await $('a[data-testid=open-registration-form-button]');
+        await createAccLocator.click();
+        await browser.pause(10000);
+
+        //3. Verify current date is displayed in the birthdate dropdowns.
+
+        //const DOBlocator = await $('span[data-type=selectors]')
+        const DOBlocator = await $('//select[@title="Month"]/option[@selected="1"]')
```

Comment:
Suggestion for next time
DOBlocator => monthDropdown
DOBlocator1 => dayDropdown
DOBlocator2 => yearDropdown